t/t-clone.sh

```diff
@@ -665,3 +665,94 @@ begin_test "clone bare empty repository"
   fi
 )
 end_test
+
+begin_test "clone (HTTP server/proxy require cookies)"
+(
+  set -e
+
+  # golang net.http.Cookie ignores cookies with IP instead of domain/hostname
+  GITSERVER_SAVED="$GITSERVER"
+  GITSERVER=$(echo "$GITSERVER" | sed 's/127\.0\.0\.1/localhost/')
```

Comment:
This is in a subshell, so you need not save the old version and restore it.  Any variables you modify here are not persisted in other tests.

Reply:
This should be fixed now.